TestClientDescribeRingIntegration is flaky

Example logs from failed run:
```
2024-05-16T09:51:35.2181052Z === RUN   TestClientDescribeRingIntegration
2024-05-16T09:51:35.2210409Z 09:51:35.220	[35mDEBUG[0m	HTTP	{"host": "192.168.200.11:10001", "method": "GET", "uri": "/agent/node_info", "duration": "2ms", "status": 200, "bytes": 538}
2024-05-16T09:51:35.2250509Z 09:51:35.224	[35mDEBUG[0m	HTTP	{"host": "192.168.200.12:10001", "method": "GET", "uri": "/agent/node_info", "duration": "3ms", "status": 200, "bytes": 538}
2024-05-16T09:51:35.2282279Z 09:51:35.227	[35mDEBUG[0m	HTTP	{"host": "192.168.200.13:10001", "method": "GET", "uri": "/agent/node_info", "duration": "3ms", "status": 200, "bytes": 538}
2024-05-16T09:51:35.2315308Z 09:51:35.231	[35mDEBUG[0m	HTTP	{"host": "192.168.200.21:10001", "method": "GET", "uri": "/agent/node_info", "duration": "3ms", "status": 200, "bytes": 538}
2024-05-16T09:51:35.2350255Z 09:51:35.234	[35mDEBUG[0m	HTTP	{"host": "192.168.200.22:10001", "method": "GET", "uri": "/agent/node_info", "duration": "3ms", "status": 200, "bytes": 538}
2024-05-16T09:51:35.2386256Z 09:51:35.238	[35mDEBUG[0m	HTTP	{"host": "192.168.200.23:10001", "method": "GET", "uri": "/agent/node_info", "duration": "3ms", "status": 200, "bytes": 538}
2024-05-16T09:51:35.2580013Z 09:51:35.257	[35mDEBUG[0m	HTTP	{"host": "192.168.200.11:10001", "method": "GET", "uri": "/storage_service/keyspaces?replication=tablets", "duration": "2ms", "status": 200, "bytes": 2}
2024-05-16T09:51:35.2586495Z 09:51:35.258	[35mDEBUG[0m	HTTP	{"host": "192.168.200.11:10001", "method": "GET", "uri": "/storage_service/keyspaces?replication=vnodes", "duration": "0ms", "status": 200, "bytes": 126}
2024-05-16T09:51:39.2982673Z 09:51:39.297	[35mDEBUG[0m	HTTP	{"host": "192.168.200.11:10001", "method": "GET", "uri": "/storage_service/keyspaces?replication=tablets", "duration": "0ms", "status": 200, "bytes": 2}
2024-05-16T09:51:39.2986397Z 09:51:39.298	[35mDEBUG[0m	HTTP	{"host": "192.168.200.11:10001", "method": "GET", "uri": "/storage_service/keyspaces?replication=vnodes", "duration": "0ms", "status": 200, "bytes": 136}
2024-05-16T09:51:39.3027204Z 09:51:39.302	[35mDEBUG[0m	HTTP	{"host": "192.168.200.11:10001", "method": "GET", "uri": "/storage_service/describe_ring/test_ks", "duration": "4ms", "status": 200, "bytes": -1}
2024-05-16T09:51:44.5917942Z 09:51:44.591	[35mDEBUG[0m	HTTP	{"host": "192.168.200.11:10001", "method": "GET", "uri": "/storage_service/keyspaces?replication=tablets", "duration": "1ms", "status": 200, "bytes": 11}
2024-05-16T09:51:44.5921698Z 09:51:44.591	[35mDEBUG[0m	HTTP	{"host": "192.168.200.11:10001", "method": "GET", "uri": "/storage_service/keyspaces?replication=vnodes", "duration": "0ms", "status": 200, "bytes": 126}
2024-05-16T09:51:44.5927376Z 09:51:44.592	[35mDEBUG[0m	HTTP	{"host": "192.168.200.11:10001", "method": "GET", "uri": "/storage_service/describe_ring/test_ks?table=test_tab", "duration": "0ms", "status": 200, "bytes": -1}
2024-05-16T09:51:47.4985513Z     client_scylla_integration_test.go:130: Cannot add already existing table "test_tab" to keyspace "test_ks"
2024-05-16T09:51:47.4990100Z --- FAIL: TestClientDescribeRingIntegration (12.28s)
```

--- a/pkg/scyllaclient/client_scylla_integration_test.go
+++ b/pkg/scyllaclient/client_scylla_integration_test.go
@@ -7,6 +7,7 @@ package scyllaclient_test
 
 import (
 	"context"
+	"fmt"
 	"net"
 	"net/http"
 	"strings"
@@ -75,17 +76,20 @@ func TestClientStatusIntegration(t *testing.T) {
 
 func TestClientDescribeRingIntegration(t *testing.T) {
 	testCases := []struct {
+		name            string
 		replicationStmt string
 		replication     scyllaclient.ReplicationStrategy
 		rf              int
 		dcRF            map[string]int
 	}{
 		{
+			name:            "simple_4",
 			replicationStmt: "{'class': 'SimpleStrategy', 'replication_factor': 4}",
 			replication:     scyllaclient.SimpleStrategy,
 			rf:              4,
 		},
 		{
+			name:            "network_topology_1",
 			replicationStmt: "{'class': 'NetworkTopologyStrategy', 'dc1': 1}",
 			replication:     scyllaclient.NetworkTopologyStrategy,
 			rf:              1,
@@ -94,6 +98,7 @@ func TestClientDescribeRingIntegration(t *testing.T) {
 			},
 		},
 		{
+			name:            "network_topology_2_2",
 			replicationStmt: "{'class': 'NetworkTopologyStrategy', 'dc1': 2, 'dc2': 2}",
 			replication:     scyllaclient.NetworkTopologyStrategy,
 			rf:              4,
@@ -103,6 +108,7 @@ func TestClientDescribeRingIntegration(t *testing.T) {
 			},
 		},
 		{
+			name:            "network_topology_3_3",
 			replicationStmt: "{'class': 'NetworkTopologyStrategy', 'dc1': 3, 'dc2': 3}",
 			replication:     scyllaclient.NetworkTopologyStrategy,
 			rf:              6,
@@ -123,29 +129,34 @@ func TestClientDescribeRingIntegration(t *testing.T) {
 	ringDescriber := scyllaclient.NewRingDescriber(context.Background(), client)
 	for i := range testCases {
 		tc := testCases[i]
-		if err := clusterSession.ExecStmt("CREATE KEYSPACE test_ks WITH replication = " + tc.replicationStmt); err != nil {
-			t.Fatal(err)
-		}
-		if err := clusterSession.ExecStmt("CREATE TABLE test_ks.test_tab (a int PRIMARY KEY, b int)"); err != nil {
-			t.Fatal(err)
-		}
-		ringDescriber.Reset(context.Background())
-		ring, err := ringDescriber.DescribeRing(context.Background(), "test_ks", "test_tab")
-		if err != nil {
-			t.Fatal(err)
-		}
-		if tc.replication != ring.Replication {
-			t.Fatalf("Replication: expected %s, got %s", tc.replication, ring.Replication)
-		}
-		if tc.rf != ring.RF {
-			t.Fatalf("RF: expected %d, got %d", tc.rf, ring.RF)
-		}
-		if !maputil.Equal(tc.dcRF, ring.DCrf) {
-			t.Fatalf("DCrf: expected %v, got %v", tc.dcRF, ring.DCrf)
-		}
-		if err := clusterSession.ExecStmt("DROP KEYSPACE test_ks"); err != nil {
-			t.Fatal(err)
-		}
+		t.Run(tc.name, func(t *testing.T) {
+			if err := clusterSession.ExecStmt(fmt.Sprintf("CREATE KEYSPACE %s WITH replication = %s", tc.name, tc.replicationStmt)); err != nil {
+				t.Fatal(err)
+			}
+			defer func() {
+				if err := clusterSession.ExecStmt("DROP KEYSPACE " + tc.name); err != nil {
+					t.Fatal(err)
+				}
+			}()
+			if err := clusterSession.ExecStmt(fmt.Sprintf("CREATE TABLE %s.test_tab (a int PRIMARY KEY, b int)", tc.name)); err != nil {
+				t.Fatal(err)
+			}
+
+			ringDescriber.Reset(context.Background())
+			ring, err := ringDescriber.DescribeRing(context.Background(), tc.name, "test_tab")
+			if err != nil {
+				t.Fatal(err)
+			}
+			if tc.replication != ring.Replication {
+				t.Fatalf("Replication: expected %s, got %s", tc.replication, ring.Replication)
+			}
+			if tc.rf != ring.RF {
+				t.Fatalf("RF: expected %d, got %d", tc.rf, ring.RF)
+			}
+			if !maputil.Equal(tc.dcRF, ring.DCrf) {
+				t.Fatalf("DCrf: expected %v, got %v", tc.dcRF, ring.DCrf)
+			}
+		})
 	}
 }
 

